### DISCUSSION TITLE
Request for DNF Package Manager Support for Typst Installation on Fedora

### DISCUSSION THREAD

**Discussion by FelixSchausberger** [typst/typst#1624] [General]
I would like to request support for the DNF package manager in the installation instructions for Typst on Fedora. Currently, the instructions on the GitHub repository do not include this option. As a Fedora user, having the ability to install Typst through DNF would greatly simplify the installation process and ensure seamless updates.

I appreciate your efforts in maintaining Typst and look forward to its continued development. TIA

**Comment by alerque** [comment]
Distro packaging falls to distro package maintainers, not upstream projects. This request should be submitted to folks with packaging access to Fedora or wherever in that community new package requests are handled.

**Reply by FelixSchausberger** [reply]
Thanks, I made a request at Red Hat Bugzilla which can be found [here](https://bugzilla.redhat.com/show_bug.cgi?id=2218928).
